docs/website/docs/guides/deployment-configurations/gpu-rocm.md

```diff
@@ -51,8 +51,8 @@ Next you will need to get an IREE runtime that includes the ROCm HAL driver.
 
 Please make sure you have followed the
 [Getting started](../../building-from-source/getting-started.md) page to build
-IREE from source, then enable the experimental ROCm HAL driver with the
-`IREE_EXTERNAL_HAL_DRIVERS=rocm` option.
+IREE from source, then enable the HIP HAL driver with the
+`IREE_HAL_DRIVER_HIP` option.
```

Comment:
Err... isn't this doc for ROCm, not HIP? If we're going to include both in the long term, they should both be documented (including when to use each, what the tradeoffs are, etc.). If we're only keeping one of them, we should go through and update the document title and other text consistently.

Reply:
That was my bad. It makes sense to create a new section for HIP, let me create a new patch